### PR TITLE
Assignment 4

### PR DESCRIPTION
**Pull request by RakshaCode**

### CODE REVIEW

File: Assignment4/RakshaCode-190165/task06.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+# Visualize the results
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+print("Task 6.5")
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((vcard.Works, RDF.type, RDFS.Datatype))
+g.add((ns.JohnSmith, vcard.Works, ns.UPM))
```
Comment (dgarijo) [inline Assignment4/RakshaCode-190165/task06.py]: vcard.Works does not exist. Create it your own ns

File: Assignment4/RakshaCode-190165/task07.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+print("TASK 7.2 SPARQL")
+q1 = """
+SELECT ?person WHERE {
+  ?person rdf:type ns:Person .
+}
+"""
+q2 = """
+SELECT ?personSubClass WHERE {
+  ?subClass rdfs:subClassOf ns:Person .
```
Comment (dgarijo) [inline Assignment4/RakshaCode-190165/task07.py]: use * to chain subClassOf properties

File: Assignment4/RakshaCode-190165/task07.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  ?person rdf:type ns:Person .
+  ?person ?pred ?obj .
+}
+"""
+q2 = """
+SELECT ?personSubClass ?pred ?obj WHERE {
+  ?subClass rdfs:subClassOf ns:Person .
+  ?personSubClass rdf:type ?subClass .
+  ?personSubClass ?pred ?obj .
+}
```
Comment (dgarijo) [inline Assignment4/RakshaCode-190165/task07.py:96]: same as above

**Review by dgarijo — CHANGES_REQUESTED** [review]
minimal changes requested

**Comment by RakshaCode** [comment]
@dgarijo Thanks for the review

**Pull request merged**